cassandra/src/test/resources/model.json

```diff
@@ -24,6 +24,7 @@
       "factory": "org.apache.calcite.adapter.cassandra.CassandraSchemaFactory",
       "operand": {
         "host": "localhost",
+        "port": 9142,
```

Comment:
Is it possible to avoid hardcoding port number  ?

Reply:
I don't see hardcoding the port number as a problem. It's defined by cassandra-unit and isn't going to change except possibly when the dependency is updated in the future. It would be a pain to refactor to avoid this.